backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -119,6 +127,37 @@ public void testGetTokensForShop_success() throws Exception {
                 inMemoryVirtualQueueRepository.getShopIdToListOfTokensMap().get(shopId).size());
     }
 
+    @Test
+    public void testGetAllShops() throws Exception {
+        // Arrange
+        UUID shopId = addShopToRepository();
+        addTokenListToShop(shopId);
+
+        // Act
+        List<ShopInfo> expectedGetAllShopsResponseList = inMemoryVirtualQueueRepository.getAllShops();
+
+        // Assert
+        assertEquals("Size of shopInfo list", expectedGetAllShopsResponseList,
+                inMemoryVirtualQueueRepository
+                        .getShopMap().values().stream().map(shop -> new ShopInfo(shop, inMemoryVirtualQueueRepository
+                                .getShopIdToListOfTokensMap().get(shop.getShopId()).stream().count()))
+                        .collect(Collectors.toList()));
+    }
+
+    @Test
+    public void testGetShopName() throws Exception {
```

Comment:
Nit. change name to testGetShopName_success.

Reply:
done